go.mod

```diff
@@ -62,6 +62,7 @@ require (
 	github.com/libp2p/zeroconf/v2 v2.2.0
 	github.com/lorenzodonini/ocpp-go v0.16.1-0.20220927200117-d4afed542168
 	github.com/lunixbochs/struc v0.0.0-20200707160740-784aaebc1d40
+	github.com/mabunixda/wattpilot v1.0.4
```

Comment:
Needs to be bumped to v1.0.5 to incorporate the license.

Reply:
bzw 1.1.1 nun schon